torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -248,6 +255,103 @@ def _should_disable_saved_tensors_hooks():
     return False
 
 
+# Custom ops that intentionally mutate inputs (e.g., in-place fusion ops)
+# and should skip the aliasing constraint check
+_CUSTOM_OP_ALIASING_ALLOWLIST = [
+    torch.ops.mkldnn._convolution_pointwise_.binary,
```

Comment:
what's the actual problem with this op? It looks like it correctly marks the input as mutating in its schema: https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/mkldnn/RegisterMkldnnOpContextClass.cpp#L53

Reply:
@tugsbayasgalan you should allow aliasing if the schema says aliasing is allowed